From edfe545d39f971eec8a93a42d4e850f510e6ae82 Mon Sep 17 00:00:00 2001
From: junjuew <junjuew@cs.cmu.edu>
Date: Mon, 26 Aug 2019 21:56:09 -0400
Subject: [PATCH] [cvat] add prefix to cvat annotation page

---
 cvat/apps/engine/urls.py | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/cvat/apps/engine/urls.py b/cvat/apps/engine/urls.py
index 5ec6d3f..e168d47 100644
--- a/cvat/apps/engine/urls.py
+++ b/cvat/apps/engine/urls.py
@@ -32,7 +32,9 @@ router.register('plugins', views.PluginViewSet)
 
 urlpatterns = [
     # Entry point for a client
-    path('', views.dispatch_request),
+    # junjuew: add url prefix for cvat annotation page
+    # path('', views.dispatch_request),
+    path('cvat/', views.dispatch_request),
 
     # documentation for API
     path('api/swagger.<slug:format>$', schema_view.without_ui(cache_timeout=0), name='schema-json'),
-- 
2.21.0

