From 989d68ef70505266dfff3a0f14c231ecd615a039 Mon Sep 17 00:00:00 2001
From: junjuew <junjuew@cs.cmu.edu>
Date: Tue, 27 Aug 2019 15:57:23 -0400
Subject: [PATCH] [cvat] make cvat mod to work within tpod

---
 cvat/apps/engine/urls.py | 8 ++++++--
 1 file changed, 6 insertions(+), 2 deletions(-)

diff --git a/cvat/apps/engine/urls.py b/cvat/apps/engine/urls.py
index 2d8122e..3585cff 100644
--- a/cvat/apps/engine/urls.py
+++ b/cvat/apps/engine/urls.py
@@ -32,7 +32,9 @@ router.register('plugins', views.PluginViewSet)
 
 urlpatterns = [
     # Entry point for a client
-    path('', views.dispatch_request),
+    # path('', views.dispatch_request),
+    # junjuew: add cvat url prefix to differentiate
+    path('cvat/', views.dispatch_request),
 
     # documentation for API
     path('api/swagger.<slug:format>$', schema_view.without_ui(cache_timeout=0), name='schema-json'),
@@ -40,5 +42,7 @@ urlpatterns = [
     path('api/docs/', schema_view.with_ui('redoc', cache_timeout=0), name='schema-redoc'),
 
     # entry point for API
-    path('api/v1/', include((router.urls, 'cvat'), namespace='v1'))
+    # path('api/v1/', include((router.urls, 'cvat'), namespace='v1'))
+    # junjuew: drf is having trouble resolving urls with namespace
+    path('api/v1/', include((router.urls)))
 ]
-- 
2.21.0

